### ISSUE TITLE
Column selection for TextBox

### ISSUE THREAD

**Issue by KvanTTT** [AvaloniaUI/Avalonia#752]
It would be great to implement Column selection in TextBox control (Similary to Shift+Alt in Visual Code or Alt in Notepad++).

**Comment by 0xFireball** [comment]
Do you mean like the multiple cursor feature?

**Comment by jogibear9988** [comment]
it means selection like here: https://raw.githubusercontent.com/bigfive/atom-sublime-select/master/screenshot.png

**Comment by DraTeots** [comment]
BTW, since WPF and Winforms hosting are implemented, while this feature is not implemented, one could use FastColoredTextBox. 
https://github.com/PavelTorgashov/FastColoredTextBox

It implements this feature as many other editor features implemented in atom, sublime etc.

**Comment by KvanTTT** [comment]
WPF and Winforms hosting is not a cross-platform solution.

**Comment by DraTeots** [comment]
FastColoredTextBox is winforms control which, as I remember, can work at least on mono on Mac and Linux. Not sure about phones. Not sure if Avalonia Winforms hosting implementation breaks cross-platformism.

**Comment by Gillibald** [comment]
I don't think this should be added to TextBox. TextBox's selection is index based. I think RichTextBox is a better fit to add such a feature.
